src/core/celery.py

```
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, unicode_literals
+import os
+from celery import Celery
+
+
+from projects.tasks.fetch_favicon import fetch_favicon
```

Ruff

`projects.tasks.fetch_favicon.fetch_favicon` imported but unused

Remove unused import: `projects.tasks.fetch_favicon.fetch_favicon`

(F401)